sound/soc/intel/boards/sdw_rt711_rt1308_rt715.c

```diff
@@ -25,6 +25,7 @@
 #include <sound/soc.h>
 #include <sound/soc-acpi.h>
 #include "../../codecs/hdac_hdmi.h"
+#include "../../codecs/rt1308.h"
```

Comment:
do you really need this include and why?

Reply:
to support 1308 in I2S mode, we need to set RT1308_PLL_S_MCLK defined in rt1308.h  in rt1308_hw_params.  please check https://github.com/thesofproject/linux/blob/upstream/soundwire/sound/soc/intel/boards/sdw_rt711_i2s_rt1308.c#L28

Reply:
ok